testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CLIWrapper.java

```diff
@@ -93,12 +94,26 @@ public CLIWrapper(boolean connect, String cliAddress) throws CliInitializationEx
      * @param consoleInput input stream to use for sending to the CLI, or {@code null} if the standard input stream should be used
      */
     public CLIWrapper(boolean connect, String cliAddress, InputStream consoleInput) throws CliInitializationException {
+        this(connect, cliAddress, consoleInput, CONNECTION_TIMEOUT);
```

Comment:
Do you want to supersede the CLI defaults? You could not provide any connectionTimeout if a negative connectionTimeout is provided.